tests/correct/globalVariables.rock

```diff
@@ -14,3 +14,11 @@ Give back X plus GlobalTwo
 
 put 6 into GlobalTwo
 say MyFuncTwo taking 2
+
+SetGlobal takes X
+put X into Global
+Give back X
```

Comment:
This should read `Give back Global` if you want it to return 11. `X` is 1 at this point

Reply:
Agreed